.github/workflows/backend-pr-approver.yml

```diff
@@ -1,13 +1,31 @@
 name: Backend PR Approver
 on:
-  workflow_run:
-    workflows: 
-      - "Backend PR Labeler"
-    types: [completed]
+  pull_request:
+    types: [labeled, unlabeled]
 jobs:
-  say-hi:
+  check-approval-labels:
+    name: Confirm Backend Approval
+    if: github.event.label.name == 'require-backend-approval' || github.event.label.name == 'ready-for-backend-review'
     runs-on: ubuntu-latest
     steps:
-      - name: Say Hi
+      - name: Check for exemption
+        id: check_exemption
         run: |
-          echo "Hi"
+          if [ ! ${{ contains(toJSON(github.event.pull_request.requested_teams.*.name), 'backend-review-group') }} ]; then
+            echo "exempt=true" >> $GITHUB_OUTPUT
+            echo "PR is exempt from backend approval."
+          else
+            echo "exempt=false" >> $GITHUB_OUTPUT
+            echo "PR requires backend approval."
+          fi
```

Comment:
Is this true? In CODEOWNERS, files either have backend-review-group listed, OR they have a lighthouse team, octo-identity, or mobile? 🤔 no... I just found [this line](https://github.com/department-of-veterans-affairs/vets-api/blob/master/.github/CODEOWNERS#L1526): `spec/lib/token_validation @department-of-veterans-affairs/lighthouse-dash @department-of-veterans-affairs/lighthouse-banana-peels @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group`
With the logic here, a PR that changes files in this dir would hit the `else` block.

Reply:
Maybe the solution is to remove `backend-review-group` from those lines in codeowners 🤔

Reply:
I think this would be an example of a CODEOWNERS issue. If they own it, our team doesn't need to be on it. If we should be involved in the review, then it'll work as expected. I think these cases will iron themselves out. I can't think of a good reason for why both would be needed yet we shouldn't be a required approver.